.github/workflows/wheels.yml

```diff
@@ -4,14 +4,26 @@ on:
   push:
     tags:
       - "*"
+    branches:
+      - main
+  pull_request:
+  workflow_dispatch:
+
+permissions:
+  contents: read # to fetch code
+  actions: write # to cancel previous workflows
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.head_ref || github.ref }}
+  cancel-in-progress: true
 
 jobs:
   build_wheels:
     name: Build wheels on ${{ matrix.os }}
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        os: ["ubuntu-22.04", "macos-12", "macos-14"]
+        os: ["ubuntu-latest", "macos-latest"]
```

Comment:
I'm fine with dropping Intel Mac, but I think there's still a macos-13 Intel runner we could use if we wanted, as long as we're okay bumping the deployment target.

(I wish there was a standard for which OS versions to support, like SPEC0 but for OS versions. If there's a standard or practice you follow, I'd be interested to know!)

Reply:
Yeah! I did get that working, but I figured it was ok to just kill it since JAX doesn't actually provide intel wheels anymore either :D. I don't feel strongly, but this seemed to be the most straightforward!

Reply:
Sounds good, following JAX here makes sense!